src/jit/importer.cpp

```diff
@@ -3607,7 +3608,33 @@ GenTreePtr Compiler::impIntrinsic(CORINFO_CLASS_HANDLE  clsHnd,
             retNode = op1;
             break;
 #endif
-
+        // Implement ByReference Ctor.  This wraps the assignment of the ref to the span field.
```

Comment:
You're correct Carol, this can be used for other byref-like fields in span like value types.  I'll expand the comment to include this.